doc/contributing.md

```diff
@@ -7,4 +7,5 @@ contributing/submitting-changes.chapter.md
 contributing/vulnerability-roundup.chapter.md
 contributing/reviewing-contributions.chapter.md
 contributing/contributing-to-documentation.chapter.md
+contributing/documentation-syntax.chapter.md
```

Comment:
I do not see much of a point moving it outside of `contributing-to-documentation` chapter. IMO, it makes sense for it to belong to that section.

If you do not like “Contributing to *this* documentation” heading, you could change it to “Contributing to Nixpkgs documentation”. But I do not think the chapter being named like that somehow denies it from also documenting the NixOS manual syntax.

Reply:
Okay. Done.